covid-vaccine-tracker/covid-vaccine-tracker.js

```diff
@@ -0,0 +1,298 @@
+const  icon = document.getElementById("icon");
+icon.onclick = function(){
+    document.body.classList.toggle("dark-mode");
+    if ( document.body.classList.contains("dark-mode")) {
+        icon.src="../assets/sun.svg";
+    }else{
+        icon.src="../assets/dark-mode.png";
+    }
+}
+
+document.getElementById("back").onclick = function () {
+  location.href = "../home/home.html";
+}
+//popup
+const button = document.querySelector("#Symptoms-button");
+const myCircule = document.getElementById("myCircule")
+button.addEventListener("click", move);
+function move() {
+  window.location.href = "../Symptoms/Symptoms.html";
+}
+const openModalButtons = document.querySelectorAll("[data-modal-target]");
+const closeModalButtons = document.querySelectorAll("[data-close-button]");
+const overlay = document.getElementById("overlay");
+
+openModalButtons.forEach((button) => {
+  button.addEventListener("click", () => {
```

Comment:
add some code to ensure that the user already had chosen a country alert him/her to do if not
also ensure that the fetch result well received 
before on click function run